src/xb-node.h

```diff
@@ -49,6 +49,19 @@ typedef enum {
 	XB_NODE_EXPORT_FLAG_LAST
 } XbNodeExportFlags;
 
+typedef struct _XbNodeIter XbNodeIter;
+
+struct _XbNodeIter
+{
+  /*< private >*/
+  gpointer	dummy1;
+  guint8	dummy2;
```

Comment:
Alignment

Reply:
Weird, this is also aligned with tabs in my editor... It just displays weirdly on Github.

Reply:
:shrug: